tests/v2/test_waku_store.nim

```diff
@@ -189,18 +187,17 @@ procSuite "Waku Store - history query":
 
     ## When
     # this query targets: pubsubtopic1 AND (contentTopic1 OR contentTopic3)    
-    let rpc = HistoryQuery(
-      contentFilters: @[HistoryContentFilter(contentTopic: contentTopic1), 
-                        HistoryContentFilter(contentTopic: contentTopic3)], 
-      pubsubTopic: pubsubTopic1
+    let req = HistoryQuery(
+      pubsubTopic: some(pubsubTopic1),
+      contentTopics: @[contentTopic1, contentTopic3] 
     )
-    let resQuery = await client.query(rpc, peer=serverPeerInfo)
+    let queryRes = await client.query(req, peer=serverPeerInfo)
 
     ## Then
     check:
-      resQuery.isOk()
+      queryRes.isOk()
 
-    let response = resQuery.tryGet() 
+    let response = queryRes.tryGet() 
```

Comment:
```suggestion
    let response = queryRes.get() 
```
Because we've already checked if the queryRes is OK?

Reply:
Yes, thx 👍🏼